test/main.sh

```diff
@@ -658,6 +658,7 @@ if [ "${1:-"all"}" != "snap" ] && [ "${1:-"all"}" != "cluster" ]; then
     run_test test_waitready "waitready"
     run_test test_vm_empty "Empty VM"
     run_test test_vm_pcie_bus "VM PCIe bus numbers"
+    run_test test_alias
```

Comment:
Could you please make that one of the first tests? This should help surface any missing cleanup bits.